packages/ensjs/src/functions/wallet/createSubname.ts

```diff
@@ -196,6 +238,26 @@ const checkCanCreateSubname = async (
     contract,
   }: Pick<BaseCreateSubnameDataParameters, 'name' | 'contract' | 'fuses'>,
 ): Promise<void> => {
+  const [resolver] = await readContract(wallet, {
+    address: getChainContractAddress({
+      client: wallet,
+      contract: 'ensUniversalResolver',
+    }),
+    abi: universalResolverFindResolverSnippet,
+    functionName: 'findResolver',
+    args: [toHex(packetToBytes(name))],
+  })
+
+  // TODO: check the interface through the resolve function
```

Comment:
Do we let the TODO, or it was done?

Reply:
comment removed and improved on the next PR